File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        //Set<Integer> st = new HashSet<>();
        PriorityQueue<Integer> pq = new PriorityQueue<>(Collections.reverseOrder());
        for(int x : nums) {
            //int l = st.size();
            //st.add(x);
            //if(l < st.size()) {
                pq.add(x);
                System.out.print(x);
            //}
        }
        while(k > 1) {
            pq.poll();
            k--;
        }
        return pq.poll();
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int>pq;
        for(int it : nums) {
            pq.push(it);
        }
        int ans;
        while(k > 0) {
            ans = pq.top();
            pq.pop();
            k--;
        }
        return ans;
    }
};